Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMigrator.codeunit.al

```diff
@@ -37,9 +37,13 @@ codeunit 40025 "GP Checkbook Migrator"
     procedure MoveTransactionsData(BankAccountNo: Code[20]; BankAccPostingGroupCode: Code[20]; CheckbookID: Text[15])
     var
         GPCheckbookTransactions: Record "GP Checkbook Transactions";
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
         PostingAccountNumber: Code[20];
         Amount: Decimal;
     begin
+        if GPCompanyAdditionalSettings.GetMigrateOnlyBankMaster() then
```

Comment:
I can go either way, but I think this check should be moved up a level. The caller should determine if the code should be run or not. Feel free to leave it as is too.